provider/hetzner.go

```diff
@@ -0,0 +1,206 @@
+/*
+	by Vladimir Smagin, 2020
+	vlad@blindage.org
+*/
```

Comment:
@21h It looks not consitent with other provider license comments:

```
/*
Copyright 2018 The Kubernetes Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```

It would great to figurate what would be the best format.